### DISCUSSION TITLE
Tailwind doesnt work in Next Link component

### DISCUSSION THREAD

**Discussion by tofuwu-dev** [vercel/next.js#48975] [Help]
Hi,

I found tailwind doesnt work in link component. i.e.
```bash
 <Link href="/link" className="text-[20px]"> Resource </Link>
```

Here, font size doesnt apply to link component.
I dont think it is tailwind configuration issue. I configured properly tailwind and installed exactly.

![image](https://user-images.githubusercontent.com/98631674/235278921-3dff99e3-7983-4ab3-9f9a-f17de898599f.png)
![image](https://user-images.githubusercontent.com/98631674/235278969-448b2d09-61f2-47e8-9a9d-5e26fbc5bfa3.png)

Hope your quick reply!

**Comment by totti-rdz** [comment]
Hi, investigating another tailwind issue I recently spin up a new next + tailwind app using create-next-app and let create-next-app install tailwind for me in the process. Perfect to test your issue I guess. Installed versions are "next": "13.3.1", "postcss": "8.4.23", "autoprefixer": "10.4.14", and "tailwindcss": "3.3.2".

I tested it with `text-5xl` and with `text-[40px]` and it worked for me. I see that the html element you are inspecting is an `<a>` tag. 
I assume that you probably have a global text-size for the `<a>` tag, which may has a higher css specificity than the text-size passed to the Link component.

Can you open up the dev tools, select the `<a>` tag and filter the styles for `font-size`. I'll attach a screenshot where I have a global style with an important font-size that'll overwrite font-sizes passed to the Link component.

<img width="912" alt="Screenshot 2023-04-29 at 14 52 35" src="https://user-images.githubusercontent.com/88311102/235303494-7fd42d56-80e9-46b0-a19f-533d79e943cc.png">

**Reply by tofuwu-dev** [reply]
Hi @totti-rdz 

Thanks for your reply!
I think I didnt use Nextjs 13 version.
So I tried to use anchor tag in Link component and I applied tailwind in a tag. Now it works. :)
